zebra-consensus/Cargo.toml

```
@@ -23,10 +23,10 @@ serde = { version = "1", features = ["serde_derive"] }
 
 futures = "0.3.17"
 futures-util = "0.3.17"
-metrics = "0.13.0-alpha.8"
+metrics = "0.17"
```

Let's use full versions, to avoid duplicate dependencies.

```suggestion
metrics = "0.17.0"
```

---

Done.